### DISCUSSION TITLE
Built in terminal enhancements

### DISCUSSION THREAD

**Discussion by JonnyTech** [doublecmd/doublecmd#1567] [Suggestions]
With the built in terminal (<kbd>CTRL</kbd>+<kbd>O</kbd>) is it possible to add:

- text selection / highlight
- copy / paste

---

Also, I notice that changing the terminal height can only be done by the mouse or in `session.ini`

```
[TfrmMain(1920x1080)]
nbConsole_Height=120
```

Can you add a keyboard option?

---

Is it possible to apply to all resolutions?

```
[TfrmMain]
nbConsole_Height=120
```

---

Is it possible to detatch the built-in terminal to its own window?

---

```
Double Commander
Version: 1.1.12 gamma
Revision: 436
Commit: c8f17e36a
Build date: 2024/04/07
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-qt5
OS version: MX 23.2 Libretto

Widgetset library: Qt 5.15.8, libQt5Pas 5.6.1
```

**Comment by alexx2000** [comment]
> Is it possible to apply to all resolutions?

No.

> Is it possible to detatch the built-in terminal to its own window?

No.
